changelog/change_add_allow_methods_and_allow_patterns_to_transaction_exit_statement.md

```diff
@@ -0,0 +1 @@
+* [#1072](https://github.com/rubocop/rubocop-rails/pull/1072): Add `AllowedMethods` and `AllowedPatterns` for `Rails/TransactionExitStatement`. ([@marocchino][])
```

Comment:
That's just my two cents. Can you rename this file to changelog/new_add_allow_methods_and_allow_patterns_to_transaction_exit_statement.md?